pkg/sql/rename_table.go

```diff
@@ -232,6 +233,29 @@ func (n *renameTableNode) startExec(params runParams) error {
 		return err
 	}
 
+	// If this table has row level ttl enabled, update the schedule_name of all
+	// row-level-ttl jobs with the name table name.
+	if tableDesc.HasRowLevelTTL() {
```

Comment:
nit: maybe extract this logic into a separate helper function. I'm ok with what it's like now though.

Reply:
I don't think there's a super clear place to put it right now. Could we save the refactor until rename table is implemented in the declarative schema changer? I think having two consumers will make the eventual "home" of this logic much more clear.